data/mainnet/tokens.js

```diff
@@ -184,6 +184,90 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'SES'],
   },
+  JJPY: {
+    chain: CHAINS_ID.MATIC_MAINNET,
+    logoUrl: './icons/jjpy.png',
+    tokenAddress: addresses.MATIC.JJPY,
+    decimals: '18',
+    vaultAddress: null,
+    priceFunction: { type: GET_PRICE_TYPES.COINGECKO_CONTRACT, params: [addresses.JPYC] },
```

Comment:
is `addresses.JPYC` correct here?

Reply:
Yes, we agreed to use this as JJPY and JCAD don't have good price feeds, so we assume their price to be equal to JPYC and CADC.